conjur/conjur_api/conjur_client.py

```diff
@@ -37,7 +37,7 @@ class ConfigException(Exception):
     use these parameters defined in this class to initialize our Python
     SDK in their code.
     """
-class Client():
+class ConjurClient():
```

Comment:
It is kind of redandunt to change the name to Conjur* because the library name is conjur. I can use like `conjur.api.Api` which is very clear to me that this is the conjur API (so as client, config etc.)

Reply:
Done

Reply:
I agree with Tamir, because in code we don't see the full package and only the Class name, so now in our testing code I only see Client and its not really readable to understand which client